.travis.yml

```diff
@@ -6,6 +6,16 @@ jobs:
         - GO111MODULE=on
       script: go test -v ./go/...
 
+    - name: "JavaScript"
+      language: node_js
+      node_js:
+       - "node"
+      install:
+       - npm --prefix ./js/bundle install
```

Comment:
There is an [`npm ci`](https://docs.npmjs.com/cli/ci.html) command that would probably be better here.